content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the Prisma Client.
+
+Example setting Prisma Client level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### NO_COLOR
+
+`NO_COLOR` if [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) will activate the `colorless` setting for error formatting and strip colors from error messages.
+
+See [Formatting via environment variables](../../concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables) for more information.
+
+### NO_PROXY
+
+`NO_PROXY` is a comma-seperated list of hostnames, IP addresses or IP ranges that do not require a proxy.
+
+```terminal
+NO_PROXY=myhostname.com,10.11.12.0/16,172.30.0.0/16
+```
+
+### HTTP_PROXY
+
+`HTTP_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTP_PROXY=http://proxy.example.com
+```
+
+### HTTPS_PROXY
+
+`HTTPS_PROXY` is set with the hostname or IP address of a proxy server.
```

Comment:
This could use explanation where this is used.

Reply:
See https://github.com/prisma/docs/pull/1422#discussion_r595185107